### DISCUSSION TITLE
Description of `simStartPars`

### DISCUSSION THREAD

**Discussion by gustavdelius** [NOAA-EDAB/ms-keyrun#79] [Q&A] [answered]
The description of the data frame `simStartPars` currently says:

"Model name and scenario, Species code, species name, length weight parameters"

I don't think that is right. I am guessing that this data frame has something to do with the initial values used when starting the Atlantis model. If that is correct, is this something we should really use when calibrating models? What for?

**Comment by sgaichas** [comment, marked as answer]
Thanks Gustav, this was actually a mistake so I have converted back to an issue, corrected the mistake in the description, and closed the issue referencing the commit with the correction. 

This data frame is intended to provide initial values from the Atlantis model to initialize estimation in the Hydra and MSSCAA models at the NEFSC (rather than Atlantis initial values). Other models may need different initial values, so we should discuss what those are and I can add them to this dataset.
